src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/entity/Objection.java

```diff
@@ -78,6 +85,8 @@ public Objection build() {
     private List<Attachment> attachments = new ArrayList<>();
     @Field("status")
     private ObjectionStatus status;
+    @Field("action_code")
+    private ActionCode actionCode;
```

Comment:
we want to store the actual number in mongo

Reply:
Changed field type.